posthog/models/organization.py

```diff
@@ -171,6 +171,12 @@ class DefaultExperimentStatsMethod(models.TextChoices):
         null=True,
         blank=True,
     )
+    default_anonymize_ips = models.BooleanField(
+        default=False,
```

Comment:
You don't need the code in lines 297, you can simply use this here (with a comment explaining the rationale)

```suggestion
        default=lambda: getattr(settings, "CLOUD_DEPLOYMENT", None) == "EU",
```

Reply:
These defaults are at Django-level so arbitrary functions are accepted here